app/scripts/storage/storage-base.js

```diff
@@ -467,8 +482,8 @@ class StorageBase {
                     this.logger.error('Error exchanging refresh token, trying to authorize again');
                     this._oauthAuthorize(callback);
                 } else {
-                    this.logger.error('Error exchanging refresh token', err);
-                    callback?.('Error exchanging refresh token');
+                    this.logger.error('Error exchanging refresh aaaaaaaaa', err);
+                    callback?.('Error exchanging refresh aaaaaaaaaaa');
```

Comment:
I think `token` made more sense? (I'm guessing this was probably just a change to make sure that it was rebuilt or something)

Reply:
Actually that's weird. Not sure why those two logs made it. They weren't even supposed to be in the commits.